hyperspy/signal.py

```diff
@@ -3093,6 +3100,11 @@ def get_dynamic_explorer_wrapper(*args, **kwargs):
 
         self._plot.plot()
 
+        # Set scale bar color:
```

Comment:
This can all be wrapped up into one call of `self._plot.scalebar_color = scalebar_color` right before plotting, regardless of whether the image is a navigator, image, etc.